volatility/cli/__init__.py

```diff
@@ -22,6 +22,11 @@
 from typing import Dict, Type, Union, Any
 from urllib import parse, request
 
+import volatility
+
+if "--live" in sys.argv:
```

Comment:
Hmm, this doesn't seem right. Why isn't this handled by the argparse logic below the --live option?

Reply:
So I concur, it's thoroughly ugly, but unfortunately the chain of imports (of `volatility.framework` upwards) would import and therefore wrap all the calls before we've set the flag.  So we've got two options:
* for every read check whether we're allowed to use the cache, and then use it or not
* wrap everything early with a wrapper that chooses whether to cache or not

From an efficiency perspective, the second option is better (I haven't timed to see how much, and chances are it's a couple lookups and an if, so should be pretty fast, but given our runs can do hundreds of reads, I went with the slight ugliness in the CLI to set the flag early enough.  I might look into a way so that change the variable late will throw an exception (so that UI developers know they need to set it *really* early, I'm just not sure how I'd go about making a variable read-only after a certain point...

Anyway, that's why it needs to be read out of sys.argv so early.  Once the parser's run, we've already imported everything to be able to enumerate it properly.